src/FixMe.jl

```diff
@@ -1,5 +1,9 @@
 module FixMe
 
 # Write your package code here.
+@warn """You've just loaded FixMe.jl. This package is not intended for productive use!
```

Comment:
Same as in the README
```suggestion
@warn """You've just loaded FixMe.jl. This package is not intended for use in production!
```